### ISSUE TITLE
Navigation breaks with custom view duration longer than dateIncrement

### ISSUE THREAD

**Issue by acerix** [fullcalendar/fullcalendar#5319]
Reported in https://fullcalendar.freshdesk.com/a/tickets/1522

Clicking "prev" doesn't work the second time and "next" should move forward by the duration but only goes 1 week.

https://codepen.io/acerix/pen/PoqyJwb?editable=true&editors=001

Works by setting `dateIncrement` to the same as `duration` manually.

**Comment by ghry5** [comment]
Did you find a workaround?
I am having the exact problem with any `dateIncrement` set to smaller than `duration`.

**Comment by acerix** [comment]
It seems like that is the same issue, demo with `dateIncrement > duration`:

https://codepen.io/acerix/pen/gOavVNr?editors=0010

The only workaround I could find it to replace next/prev with custom buttons.

**Comment by jorritheijn** [comment]
I'm having the same issue, can you share the code of the workaround you did?

**Comment by janswist** [comment]
> It seems like that is the same issue, demo with `dateIncrement > duration`:
> 
> https://codepen.io/acerix/pen/gOavVNr?editors=0010
> 
> The only workaround I could find it to replace next/prev with custom buttons.

So there are no plans on fixing this any soon?

**Comment by Stefmachine** [comment]
I had the same problem in my app, trying to make a custom 2 days grid (named `dayGridTwoDays`) view with only 1 day increment.

Here is my temporary fix with custom button, keeping native functionality with multiple views allowed.
```
    customButtons: {
        prev: {
            click: function() {
                var api = FullCalendar.getApi();
                if(api.view.type === 'dayGridTwoDays'){
                    api.changeView('dayGrid'); //Switch to one day view
                    api.prev(); //Set to previous day
                    api.changeView('dayGridTwoDays'); //Switch back to 2 days view
                }
                else{
                    api.prev();
                }
            }
        },
        next: {
            click: function() {
                var api = FullCalendar.getApi();
                if(api.view.type === 'dayGridTwoDays'){
                    api.changeView('dayGrid'); //Switch to one day view
                    api.next(); //Set to next day
                    api.changeView('dayGridTwoDays'); //Switch back to 2 days view
                }
                else{
                    api.next();
                }
            }
        }
    }
```

Please note that I am using Vue adapter so `FullCalendar.getApi();` is actually `vm.$refs.fullCalendar.getApi();`.

**Comment by fabianosanttana** [comment]
Any update for this issue?

**Comment by dhruvaraj-jaiswal** [comment]
prev and next navigation is not working for me as well :(

**Comment by acerix** [comment]
For reference, here's a workaround for a year-long view with 6 month increment:

https://codepen.io/acerix/pen/vYgMxvO?editors=001

**Comment by arshaw** [comment]
this fix has been released in [v5.8.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v5.8.0)

upgraded repro:
https://codepen.io/arshaw/pen/WNpLBPO?editors=0010
